zeppelin-web/README.md

```diff
@@ -19,19 +19,27 @@ It is recommended to install node 6.0.0+ since Zeppelin uses 6.9.1+ (see [creati
 
 All build commands are described in [package.json](./package.json)
 
-```sh
 # install required depepdencies and bower packages (only once)
-$ npm install -g yarn
-$ yarn install
+```
+$ npm install -g yarn bower npm-run-all rimraf grunt webpack karma karma-phantomjs-launcher
```

Comment:
@Leemoonsoo  Please try build project at clear box (without reinstalled nodejs and npm), and you got a lot of failures at zeppelin-web. And try my patch...